sdk/translation/azure-ai-translation-document/src/test/java/com/azure/ai/translation/document/DocumentFilterTests.java

```diff
@@ -0,0 +1,276 @@
+import com.azure.core.util.polling.SyncPoller;
+import java.time.format.DateTimeFormatter;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class DocumentFilterTests extends DocumentTranslationClientTestBase {
+
+    @LiveOnly
+    @RecordWithoutRequestBody
+    @Test
+    public void testGetDocumentStatusesFilterByStatus() {
+        // create translation job
+        TranslationStatus translationStatus = createSingleTranslationJob(2);
+
+        // Add Status filter
+        List<String> succeededStatusList = Arrays.asList(DocumentTranslationStatus.SUCCEEDED.getValue());
+
+        RequestOptions requestOptions = new RequestOptions();
+        requestOptions.addQueryParam("statuses",
+                succeededStatusList.stream()
+                        .map(paramItemValue -> Objects.toString(paramItemValue, ""))
+                        .collect(Collectors.joining(",")),
+                false);
+
+        try {
+            PagedIterable<BinaryData> response = getDocumentTranslationClient()
+                    .getDocumentsStatus(translationStatus.getId(), requestOptions);
+            for (BinaryData d : response) {
+                String status = new ObjectMapper().readTree(d.toBytes()).get("status").asText();
+                assertTrue(succeededStatusList.contains(status));
+            }
+        } catch (Exception e) {
+            System.err.println("An exception occurred: " + e.getMessage());
+            e.printStackTrace();
+        }
+    }
+
+    @LiveOnly
+    @RecordWithoutRequestBody
+    @Test
+    public void testGetDocumentStatusesFilterByIds() {
+        // create translation job and get all the document IDs
+        TranslationStatus translationStatus = createSingleTranslationJob(2);
+        List<String> testIds = new ArrayList<>();
+        try {
+            PagedIterable<DocumentStatus> response = getDocumentTranslationClient()
+                    .getDocumentsStatus(translationStatus.getId());
+            for (DocumentStatus d : response) {
+                testIds.add(d.getId());
+            }
+        } catch (Exception e) {
+            System.err.println("An exception occurred: " + e.getMessage());
+            e.printStackTrace();
+        }
+
+        // Add id filter
+        RequestOptions requestOptions = new RequestOptions();
+        requestOptions.addQueryParam("ids", testIds.get(0), false);
+
+        try {
+            PagedIterable<BinaryData> response = getDocumentTranslationClient()
+                    .getDocumentsStatus(translationStatus.getId(), requestOptions);
+            for (BinaryData d : response) {
+                String id = new ObjectMapper().readTree(d.toBytes()).get("id").asText();
+                assertTrue(testIds.contains(id));
+            }
+        } catch (Exception e) {
+            System.err.println("An exception occurred: " + e.getMessage());
+            e.printStackTrace();
+        }
+    }
+
+    @LiveOnly
+    @RecordWithoutRequestBody
```

Comment:
no need to have @RecordWithoutRequestBody with LievOnly tags

Reply:
Tests are now Live